This patch adds Debian build version to compiler full version.

Index: fpc/fpcsrc/compiler/version.pas
===================================================================
--- fpc.orig/fpcsrc/compiler/version.pas
+++ fpc/fpcsrc/compiler/version.pas
@@ -81,6 +81,9 @@ function full_version_string:string;
 
 implementation
 
+const
+  FullVersionString={$INCLUDE version.inc};
+
 function version_string:string;
 begin
   version_string := version_nr+'.'+release_nr+'.'+patch_nr;
@@ -89,11 +92,7 @@ end;
 
 function full_version_string:string;
 begin
-  full_version_string := version_nr+'.'+release_nr+'.'+patch_nr+minorpatch
-{$ifdef REVINC}
-  +'-'+{$i revision.inc}
-{$endif REVINC}
-  ;
+  full_version_string := FullVersionString;
 end;
 
 end.
